app/views/publishers/vacancies/build/schools.html.slim

```diff
@@ -15,7 +15,7 @@
 
         = render "publishers/vacancies/vacancy_form_partials/hidden_state_input", f: f
 
-        = f.govuk_fieldset legend: { text: t("helpers.legend.publishers_job_listing_schools_form.organisation_ids") } do
+        = f.govuk_fieldset legend: { text: t("helpers.legend.publishers_job_listing_schools_form.organisation_id#{'s' if @multiple_schools}") } do
```

Comment:
Or use `count: @multiple_schools`, but this is fine too

Reply:
@multiple_schools is a boolean